thorlcr/thorutil/thormisc.hpp

```diff
@@ -606,6 +609,118 @@ extern graph_decl bool getExpertOptBool(const char *opt, bool dft=false);
 extern graph_decl __int64 getExpertOptInt64(const char *opt, __int64 dft=0);
 extern graph_decl StringBuffer &getExpertOptString(const char *opt, StringBuffer &out);
 
+////
+// IContextLogger
+class CThorContextLogger : implements IContextLogger, public CSimpleInterface
```

Comment:
trivial: more efficient to use CSimplerInterfaceOf<IContextLogger> , avoids a VMT